portfolio/src/main/java/com/google/sps/data/MessagesObject.java

```diff
@@ -0,0 +1,26 @@
+package com.google.sps.data;
+
+/** Class containing hardcoded messages. */
+public final class MessagesObject {
+
+    private String message1;
+    private String message2;
+    private String message3;
+
```

Comment:
What if you want to encode 4 or more messages? Do you add a fourth field and so on? What if there are 100 messages? This design is not very flexible. This is optional (I strongly recommend for practice) but if you want to support sending an arbitrary amount of messages I suggest using a List field instead so you can encode a list of messages like this:
```json
{
  "messages": ["foo", "bar", "baz", "far", "faz"]
}
```
Instead of necessitating at most 3 messages:
```json
{
  "message1": "foo",
  "message2": "bar",
  "message3": "baz
  // how to encode a 4th, 5th, 100th message?
}
```